go.mod

```diff
@@ -55,3 +55,5 @@ require (
 	k8s.io/klog/v2 v2.20.0 // indirect
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b // indirect
 )
+
+replace github.com/sigstore/rekor => github.com/sigstore/rekor v0.3.1-0.20211119021052-57b3fef494e0
```

Comment:
we can probably do this with a normal bump instead of the replace here.

Reply:
I think we'll need a new release here in order to include hashedrekord type!

Reply:
Ah I think you can do something like: `go get github.com/sigstore/rekor@cae63bd4e389b56e352c7ffebbb86d1c1437409b` to pin to an unreleased commit.

Reply:
Argh thank you! sorry misinterpreted what you meant by normal bump!